acos_client/v30/slb/virtual_port.py

```diff
@@ -113,12 +113,10 @@ def _set(
                 params['port']['template-tcp'] = virtual_port_templates.get('template-tcp', None)
             params['port']['template-policy'] = virtual_port_templates.get('template-policy', None)
 
-        if autosnat:
-            params['port']['auto'] = int(autosnat)
-        if ipinip:
-            params['port']['ipinip'] = int(ipinip)
-        if use_rcv_hop:
-            params['port']['use-rcv-hop-for-resp'] = int(use_rcv_hop)
+        params['port']['auto'] = int(autosnat)
```

Comment:
This breaks the functionality of POST. Users of this library shouldn't have to define non-positional arguments